static/manual-markdown.Rmd

```diff
@@ -432,4 +421,15 @@ We plan to add more documentation to ease the creation of locally specific scena
 
 </div>
 
-<script>(function () { var script = document.createElement("script"); script.type = "text/javascript"; script.src = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"; document.getElementsByTagName("head")[0].appendChild(script); })();</script>
\ No newline at end of file
+<script>(function () { var script = document.createElement("script"); script.type = "text/javascript"; script.src = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"; document.getElementsByTagName("head")[0].appendChild(script); })();</script>
+
+```{r, echo=FALSE, eval=FALSE}
+# change image links
+f = readLines("regions_www/manual.html")
+f = gsub(pattern = "/home/robin/npct/pct-shiny/www/assets/", replacement = "./assets/", f)
```

Comment:
Ah, I see.
Sorry but this isn't a proper solution, it looks far hackier than just writing html.  Is there a reason we need R code in the manual?